concepticondata/conceptlists/Castro-2015-608.tsv

```diff
@@ -396,13 +396,13 @@ Castro-2015-608-394	394	to spurt			喷	to spurt
 Castro-2015-608-395	395	to brush on			涂上（漆在棺材上）	to apply (lacquer to a coffin)
 Castro-2015-608-396	396	to sweep	230	SWEEP	扫	to sweep
 Castro-2015-608-397	397	to open (door)	1156	OPEN	开（门）	to open (a door wide)
-Castro-2015-608-398	398	to open (mouth)			张（嘴）	to open (mouth)
+Castro-2015-608-398	398	to open (mouth)	3437	OPEN (MOUTH)	张（嘴）	to open (mouth)
 Castro-2015-608-399	399	to turn off	1046	SHUT	关（门）	finished
 Castro-2015-608-400	400	to shut (eyes)	2063	CLOSE (EYES)	闭（眼睛）	to shut, close (eyes)
 Castro-2015-608-401	401	to hold in mouth			衔	to hold in one's mouth (e.g. a pipe)
 Castro-2015-608-402	402	to bite	1403	BITE	咬	to bite
 Castro-2015-608-403	403	to nod			点（头）	to nod
-Castro-2015-608-404	404	to wring			拧（毛巾）	to wring (towel)
+Castro-2015-608-404	404	to wring	3224	WRING	拧（毛巾）	to wring (towel)
```

Comment:
The mapping seems to be correct although other lists with the same gloss are mapped to TWIST or TWIST (AROUND). @mathildavz Could you open an issue so that we check the other mappings.

Reply:
One can argue that the TOWEL part is disambiguating here, as it is then clear that it is the movement by which you wrist a towel to get out the moisture (or something similar). An issue is useful, though.